tutorials/misc/binary_serialization_api.rst

```diff
@@ -100,17 +103,17 @@ of packet:
 +----------+-------+-----------+--------------------------+
 | Offset   | Len   | Type      | Description              |
 +==========+=======+===========+==========================+
-| 4        | 4     | Integer   | Signed, 32-Bit Integer   |
```

Comment:
It's possible to be 32 bit if it's godot 32 bit no ?

Reply:
https://github.com/godotengine/godot/blob/master/core/io/marshalls.cpp#L134

Reply:
I'm not completely sure, but I know that the current documentation is wrong, there's [this](https://github.com/godotengine/godot/blob/master/core/variant.h#L205), and it will definitely always be 64-bit after https://github.com/godotengine/godot/pull/21922